sqlglot/dialects/databricks.py

```diff
@@ -51,7 +51,11 @@ class Generator(Spark.Generator):
             exp.ToChar: lambda self, e: self.function_fallback_sql(e),
         }
 
-        PARAMETER_TOKEN = "$"
+        def parameter_sql(self, expression: exp.Parameter) -> str:
+            # This corresponds to "widget parameters" in Databricks, i.e. ${x}
+            # https://docs.databricks.com/en/notebooks/widgets.html#language-sql
+            this = self.sql(expression, "this")
+            return f"${{{this}}}" if expression.args.get("wrapped") else this
```

Comment:
Only Databricks hit the "wrapped" section of this method and it was added in https://github.com/tobymao/sqlglot/pull/1142/files. I didn't generate "widget params" on purpose, because I wasn't sure if this is a widely used feature & if it'd thus make sense to generate `${x}` in place of `@x`.

@dmoore247 thoughts?